CHANGELOG.md

```diff
@@ -4,6 +4,7 @@
 
 ### Changes
 
+- Rework transactions (#880) @Tyrrrz
```

Comment:
Please add a more descriptive entry such as `CaptureTransaction takes concrete Transaction class` or whatever else was done here worth noting to the users?